set.mm

```diff
@@ -150680,6 +152142,35 @@ the symbol at any position is repeated at multiples of L (modulo the
       BCWCVOVPVJVPWCQWCVPRSTVPVLHZWBVMCWDVJVPVJVLWDVJRWDQSTVAVB $.
   $}
 
+  ${
+    $d W i $.
+    $( A prefix of length 2.  (Contributed by AV, 15-May-2020.) $)
+    pfx2 $p |- ( ( W e. Word V /\ 2 <_ ( # ` W ) )
+                 -> ( W prefix 2 ) = <" ( W ` 0 ) ( W ` 1 ) "> ) $=
+      ( vi wcel c2 chash cfv wbr cc0 co c1 wceq wa cn0 a1i adantr syl cvv fveq2
+      cfzo cword cle cfz cpfx cs2 2nn0 lencl simpr elfz2nn0 syl3anbrc cv pfxlen
+      wral cpr s2len eqcomi w3a simpl cn 2nn lbfzo0 mpbir 3jca pfxfv fvex s2fv0
+      ax-mp syl6eqr clt 1nn0 1lt2 elfzo0 mpbir3an mp3an3 wb 0nn0 pm3.2i eqeq12d
+      s2fv1 ralprg mpbir2and eqeq1 oveq2 fzo0to2pr syl6eq raleqdv anbi12d mpdan
+      adantl pfxcl simp2 cr wi 1red 2re nn0re ltleletr imp 3adant1 jca elnnnn0c
+      mpani sylibr sylbi sylan2 ltletr 3impia s2cld eqwrd syl2anc mpbird syldan
+      wrdsymbcl ) BAUAZDZEBFGZUBHZEIXPUCJDZBEUDJZIBGZKBGZUEZLZXOXQMZENDZXPNDZXQ
+      XRYEYDUFOXOYFXQABUGPXOXQUHEXPUIZUJXOXRMZYCXSFGZYBFGZLZCUKZXSGZYLYBGZLZCIY
+      ITJZUMZMZYHYIELZYRABEULYHYSMZYREYJLZYOCIKUNZUMZUUAYTYJEXTYAUOUPOYTUUCIXSG
+      ZIYBGZLZKXSGZKYBGZLZYTUUDXTUUEYTXOXRIIETJZDZUQZUUDXTLYHUULYSYHXOXRUUKXOXR
+      URXOXRUHUUKYHUUKEUSDZUTEVAVBOVCPIEABVDQXTRDUUEXTLIBVEXTYARVFVGVHYHUUIYSYH
+      UUGYAUUHXOXRKUUJDZUUGYALUUNKNDZUUMKEVIHZVJUTVKKEVLVMKEABVDVNYARDUUHYALKBV
+      EXTYARVSVGVHPYTINDZUUOMZUUCUUFUUIMVOUURYTUUQUUOVPVJVQOYOUUFUUICIKNNYLILYM
+      UUDYNUUEYLIXSSYLIYBSVRYLKLYMUUGYNUUHYLKXSSYLKYBSVRVTQWAYSYRUUAUUCMVOYHYSY
+      KUUAYQUUCYIEYJWBYSYOCYPUUBYSYPUUJUUBYIEITWCWDWEWFWGWIWAWHYHXSXNDZYBXNDYCY
+      RVOXOUUSXRABEWJPYHXTYAAXRXOIIXPTJZDZXTADXRXPUSDZUVAXRYEYFXQUQZUVBYGUVCYFK
+      XPUBHZMUVBUVCYFUVDYEYFXQWKYFXQUVDYEYFXQUVDYFUUPXQUVDVKYFKWLDZEWLDZXPWLDZU
+      QZUUPXQMZUVDWMYFUVEUVFUVGYFWNUVFYFWOOXPWPVCZKEXPWQQXBWRWSWTXPXAXCXDZXPVAX
+      CIABXMXEXRXOKUUTDZYAADXRUUOUVBKXPVIHZUVLUUOXRVJOUVKXRUVCUVMYGYEYFXQUVMYEY
+      FMZUUPXQUVMVKUVNUVHUVIUVMWMYFUVHYEUVJWIKEXPXFQXBXGXDKXPVLUJKABXMXEXHXSCAY
+      BXIXJXKXL $.
+  $}
+
   $( A word of length 3 represented as triple of ordered pairs.  (Contributed
```

Comment:
3 --> three
(I may be a bit rigid... but to me, that's a bit like writing in all caps.)

Reply:
Done